### ISSUE TITLE
[ALBERT] Is finetuning to increase layers mandatory?

### ISSUE THREAD

**Issue by DomHudson** [google-research/google-research#126]
Hi! Many thanks for the interesting research!
The paper states:
> Networks with 3 or more layers are trained by fine-tuning using the parameters from the depth before (e.g., the 12-layer network parameters are fine-tuned from the checkpoint of the 6-layer network parameters).

I was wondering if this is deemed mandatory for the model to converge or was this simply a time saving mechanism to avoid starting from scratch?

Any insights would be great!
Many thanks,
Dom

**Comment by DomHudson** [comment]
The reason I ask this is because I'm finding my network suddenly plateaus early  in training. It's likely there's a bug but it would still be useful to know if you deem the finetuning step as you add layers an optimisation or necessary .

Many thanks!

![image](https://user-images.githubusercontent.com/10864294/68803502-c6799680-0657-11ea-938a-6d30f60dc247.png)

**Comment by lanzhzh** [comment]
No, this is not mandatory. Actually, it is better to train from scratch for all models except xxlarge we our results showed section 4.7 (https://arxiv.org/pdf/1909.11942.pdf) that fine-tune from lower layer network did not help. Your training loss look reasonable.

**Comment by DomHudson** [comment]
Thanks!
